### ISSUE TITLE
Should never suggest the same file to import

### ISSUE THREAD

**Issue by lencioni** [trotzig/import-js#227]
You may be working in a file called `foo.js` and have a `foo/index.js` as well, and when you import `foo`, it currently gives you the option of both, but it should definitely exclude the current file.

**Comment by trotzig** [comment]
Good point. This should be pretty easy to fix. We have to be a little but careful about comparing directly against `path_to_current_file` because I'm not sure all editor plugins pass in the same values (relative vs. absolute path).
